registry/server/templates/services/renderTemplate.spec.ts

```diff
@@ -18,13 +18,13 @@ describe('renderTemplate', () => {
                         data: `
                             <div id="include-id-1">
                                 This include has all necessary attributes
-                                and a specified link header which is a stylesheet
+                                and an empty link header which is a stylesheet
                             </div>
                         `,
                         headers: {
                             'X-Powered-By': 'JS',
                             'X-My-Awesome-Header': 'Awesome',
-                            'Link': 'https://my.awesome.server/my-awesome-stylesheet.css;rel=stylesheet;loveyou=3000',
+                            'Link': '',
```

Comment:
Add new request instead of modification of the existing one.
Also test that not only empty `Link` but also absence of `Link` works